src/main/java/seedu/address/model/project/Participation.java

```diff
@@ -33,19 +33,65 @@ public Participation(Person person, Project project) {
     }
 
     /**
-     * indicates attendance for the meeting
+     * Alternative constructor that allows specifying the role of the person
+     */
+    public Participation(Person person, Project project, Role role) {
+        this.person = person;
+        this.project = project;
+        this.role = role;
+        tasks = new HashSet<>();
+        meetings = new HashSet<>();
+    }
```

Comment:
Good implementation of the Association class, we'll see how this plays out when we build the subsequent features.